records_versioning/models.py

```
@@ -6,7 +6,7 @@ class RecordVersion(models.Model):
     id = models.BigAutoField(primary_key=True)
     resource_type = models.CharField(max_length=48)  # 'Patient','Encounter','LabResult','MedicationOrder'
 
-    resource_id = models.UUIDField()
+    resource_id = models.PositiveIntegerField()  # Changed from UUIDField to PositiveIntegerField
```

Ruff (0.12.2)

Line too long (97 > 88)

(E501)